framework/moose.mk

```diff
@@ -219,17 +219,17 @@ clobberall:: clobber
 
 compile_commands_all_srcfiles := $(moose_srcfiles) $(srcfiles)
 compile_commands.json::
-	ifeq (4.0,$(firstword $(sort $(MAKE_VERSION) 4.0)))
+ifeq (4.0,$(firstword $(sort $(MAKE_VERSION) 4.0)))
 		$(file > .compile_commands.json,$(CURDIR))
 		$(file >> .compile_commands.json,$(libmesh_CXX))
 		$(file >> .compile_commands.json,$(libmesh_CPPFLAGS) $(libmesh_CXXFLAGS) $(app_INCLUDES) $(libmesh_INCLUDE) $(ADDITIONAL_INCLUDES))
 		$(file >> .compile_commands.json,$(compile_commands_all_srcfiles))
-	else
+else
 		@echo $(CURDIR) > .compile_commands.json
 		@echo $(libmesh_CXX) >> .compile_commands.json
 		@echo $(libmesh_CPPFLAGS) $(libmesh_CXXFLAGS) $(app_INCLUDES) $(libmesh_INCLUDE) $(ADDITIONAL_INCLUDES) >> .compile_commands.json
 		@echo $(compile_commands_all_srcfiles) >> .compile_commands.json
-	endif
+endif
 	@ $(FRAMEWORK_DIR)/scripts/compile_commands.py < .compile_commands.json > compile_commands.json
 	@rm .compile_commands.json
```

Comment:
Cahn you please double check that these cleanup statements after the `endif`, which belong to the `compile_commands.json::` target are indented correctly?

Reply:
Looks like they are though

Reply:
I have updated the indentation of everything to match. They seem to be cosmetic changes only, though.
The cleanup commands do run during the `compile_commands.json::` recipe. I also verified that they are not hit during the regular build recipe.